src/util/HandleSocketMessage.ts

```diff
@@ -21,14 +22,15 @@ export default class HandleSocketMessage {
                public currentElement: CanvasElement,
                public tools: Tool[],
                public layer: Konva.Layer,
-               public canvasElements: CanvasElement[]) {
+               public canvasElements: CanvasElement[],
+               public stageEvent: CustomStageEvent) {
   }
 
   handle = (): SocketHandlerCallback | undefined => {
     if (!this.checkIsUndefined()) {
       if (this.canvasElement.layerId !== this.currentElement.layerId) {
         if (this.canvasElement.tool.temporary) {
-          this.renderCanvas(this.tools, this.canvasElement, this.layer)
+          this.renderCanvas(this.tools, this.canvasElement, this.layer, this.stageEvent)
```

Comment:
this.stageEvent refers to the current stageconfigurations, using it to format the global positions